graders/r/README.md

```diff
@@ -19,7 +19,7 @@ used for the course.
 
 See the Dockerfile for full details but we include
 
-- tidyverse and data.table for data manipulation;
+- tidyverse and `data.table` for data manipulation;
 - RUnit, testthat and tinytest for testing;
 - microbenchmark, rbenchmark and bench for timing;
 - shiny, flexdashboard, dygraphs for shiny;
```

Comment:
Should more of these also get backticks?

Reply:
Changed in [f2429b4](https://github.com/PrairieLearn/PrairieLearn/pull/11643/commits/f2429b43439d5e25b6eb82f254e7d55f94d69384).